frontend/src/app/components/negotium-task/negotium-task-details/task-details/task-details.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="task-details">
+  <div class="tasks-paragraphs">
+    <p>Status: {{ task.status }}</p>
+    <p>Estimated time:</p>
+    <p>Assigned to: {{ task.assignedTo }}</p>
+    <p>Importance level: {{ task.importanceLevel }}</p>
+    <p>Category: {{ task.category }}</p>
+  </div>
+  <div class="control-buttons">
+    <button class="button-yellow">Edit task</button>
+    <button [ngClass]="task.returnButtonColor()">Change Status</button>
```

Comment:
tu można zrobić jakieś gettery ale to później jak już nie będzie tego sztucznego obiektu